src/Theme/PageTemplate.elm

```diff
@@ -30,6 +30,9 @@ view pageInfo =
         [ HeaderTemplate.view { content = t pageInfo.title }
         , main_ [ css [ mainStyle ] ]
             [ button [ onClick LanguageChangeRequested ] [ text (t ChangeLanguage) ]
+            , p [] [ text (t WelcomeP1) ]
```

Comment:
@aaaaargZombies @r-ferrier Should we stick a container around these `p`'s?

Reply:
maybe a container with something like 

```css
.center {
  box-sizing: content-box;
  max-inline-size: 60ch;
  margin-inline: auto;
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

if we're getting into layout now, but maybe beyond scope of ticket

Reply:
Feels out of scope to me?

Reply:
I want the foundational layout, markup and accessibility to be addressed as we add stuff here, even though we have no theme as yet. Just the basics like containers so responsiveness is by default.